tests/cypress/e2e/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -282,9 +282,9 @@ context('Review pipeline feature', () => {
             cy.get('.cvat-issues-sidebar-previous-frame')
                 .should('have.attr', 'style')
                 .and('contain', 'opacity: 0.5;'); // the element is not active
-            cy.get('.cvat-issues-sidebar-next-frame').should('be.visible').click();
+            cy.get('.cvat-issues-sidebar-next-frame').should('be.visible').click({ force: true });
             cy.checkFrameNum(1);
-            cy.get('.cvat-issues-sidebar-next-frame').should('be.visible').click();
+            cy.get('.cvat-issues-sidebar-next-frame').should('be.visible').click({ force: true });
```

Comment:
Why do we need this change? If we force the clicks it prevents cypress from checking if the button is [actionable](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Forcing), which is not what we really want here.

Reply:
@klakhov sure I'll add a cypress test

Reply:
`it('Calculates frames correctly based on issues and conflicts', () => {
    // Mock state to simulate Redux store
    const state = {
        annotation: {
            player: {
                frame: {
                    number: 3 // Current frame number
                }
            },
            workspace: Workspace.REVIEW,
            canvas: {
                ready: true,
                activeControl: ActiveControl.CURSOR
            }
        },
        review: {
            issues: [
                { id: 1, frame: 1, resolved: false },
                { id: 2, frame: 3, resolved: true },
                { id: 3, frame: 5, resolved: false }
            ],
            conflicts: [
                { id: 4, frame: 2 },
                { id: 5, frame: 4 }
            ]
        },
        settings: {
            shapes: {
                showGroundTruth: true
            }
        },
        // Other required state properties...
    };

    cy.wrap(state).then((state) => {
        cy.window().its('store').invoke('replaceReducer', () => state);

        // Ensure frames are calculated correctly
        cy.get('@dynamicLeftProps').then((dynamicLeftProps) => {
            const onClick = dynamicLeftProps.onClick;
            const nearestLeftFrame = onClick.toString().match(/\d+/g)[0];
            expect(nearestLeftFrame).to.equal('2'); // Nearest left frame should be 2
        });

        cy.get('@dynamicRightProps').then((dynamicRightProps) => {
            const onClick = dynamicRightProps.onClick;
            const nearestRightFrame = onClick.toString().match(/\d+/g)[0];
            expect(nearestRightFrame).to.equal('4'); // Nearest right frame should be 4
        });
    });
});
`

Is this test okay @klakhov ?

Reply:
@thekavikumar,
No, its not what we expect here. Consider looking on [review pipeline test](https://github.com/opencv/cvat/blob/develop/tests/cypress/e2e/actions_users/registration_involved/case_28_review_pipeline_feature.js). We use cypress to test UI from user perspective. Which means, we dont emulate state nor check components props. You can extend the review test with resolving some issues,setting hide resolved issues and check that navigation is correct as the normal user would do. We already test basic issue filter navigation on [280 line and below](https://github.com/opencv/cvat/blob/develop/tests/cypress/e2e/actions_users/registration_involved/case_28_review_pipeline_feature.js#L280)